mitosheet/src/utils/code.tsx

```diff
@@ -131,15 +131,19 @@ export function isMitoAnalysisCode(codeText: string): boolean {
     Returns true if the cell contains a mitosheet.sheet(analysis_to_replay={analysisName})
 */
 export function containsMitosheetCallWithSpecificAnalysisToReplay(codeText: string, analysisName: string): boolean {
-    return codeText.includes('sheet(') && codeText.includes(`analysis_to_replay="${analysisName}"`)
+    // Remove any whitespace from codeText
+    const codeTextCleaned = codeText.replace(/\s/g, '');
+    return codeTextCleaned.includes('sheet(') && codeTextCleaned.includes(`analysis_to_replay="${analysisName}"`)
 }
 
 
 /* 
     Returns true if the cell contains a mitosheet.sheet(analysis_to_replay={analysisName})
 */
 export function containsMitosheetCallWithAnyAnalysisToReplay(codeText: string): boolean {
-    return isMitosheetCallCode(codeText) && codeText.includes(`analysis_to_replay=`)
+    // Remove any whitespace from codeText
+    const codeTextCleaned = codeText.replace(/\s/g, '');
+    return isMitosheetCallCode(codeTextCleaned) && codeTextCleaned.includes(`analysis_to_replay=`)
```

Comment:
I think we should call the isMitosheetCallCode on the original text here - so we aren't making assumptions on how it handles things.